src/components/ListSelectionPage/ListSelectionPage.js

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import { useIntl } from 'react-intl';
+import { InjectAppServices } from '../../services/pure-di';
+import { Loading } from '../Loading/Loading';
+import { SubscriberListState } from '../../services/shopify-client';
+import { Promotional } from '../ContactPolicy/Promotional';
+import InfiniteScroll from 'react-infinite-scroll-component';
+import { useFetch } from '../../hooks/useFetch';
+
+const listsPerPage = 20;
+
+const ListRow = ({ list }) => {
+  return (
+    <tr>
+      <td aria-label="List Check">
+        <label className="dp-list-check">
+          <input type="checkbox" />
+          <span className="checkmark" />
+        </label>
+      </td>
+      <td aria-label="Name">
+        <span>{list.name}</span>
+      </td>
+      <td aria-label="Subscribers Amount">
+        <span>{list.amountSubscribers}</span>
+      </td>
+    </tr>
+  );
+};
+
+const ListTable = ({ lists }) => {
+  const intl = useIntl();
+  const _ = (id, values) => intl.formatMessage({ id: id }, values);
+
+  const rows = [];
+  lists.forEach((list) => {
+    rows.push(<ListRow list={list} key={list.id} />);
+  });
+  return (
+    <table
+      className="dp-c-table"
+      aria-label="Selección de listas a excluir de la regla"
+      summary="Selección de listas a excluir de la regla"
+    >
+      <thead>
+        <tr>
+          <th>
+            <img
+              src={_('common.ui_library_image', { imageUrl: 'check-grey.svg' })}
+              alt="check list"
+            />
+          </th>
+          <th>Nombre de la lista</th>
+          <th>suscriptores</th>
+        </tr>
+      </thead>
+      <tbody>{rows}</tbody>
+    </table>
+  );
+};
+
+export const ListSelectionPage = InjectAppServices(({ dependencies: { dopplerApiClient } }) => {
```

Comment:
is this a new view or just a component to be used in another view? the page suffix doesn't know if it's okay here. In any case, ListSelectionPage is a very general name for the list of subscribers

Reply:
It is a component to be used in another view. We can name it 'SubscriberListSelector'.

Reply:
I would think that you should call `SubscriberLists` and receive a prop that indicates the opening mode: if it is selectable or not. What do you think ? because this list could be used in another view.